httpobs/scanner/analyzer/content.py

```diff
@@ -12,7 +12,7 @@
 
 
 # Compat between Python 3.4 and Python 3.5 (see: https://github.com/mozilla/http-observatory-website/issues/14)
-if not hasattr(json, 'JSONDecodeError'):
+if not hasattr(json, 'JSONDecodeError'):  # pragma: no cover
```

Comment:
You should be able to mock this in tests by using `side_effect`. That would allow you to remove the `pragma` and this very non-Pythonic check (i.e. ask for forgiveness rather than permission), replacing with a try/except instead.